quaint/src/connector/mysql/wasm/error.rs

```diff
@@ -1,22 +1,21 @@
 use crate::error::{DatabaseConstraint, Error, ErrorKind};
-use mysql_async as my;
+use thiserror::Error;
 
+#[derive(Debug, Error)]
+enum MysqlAsyncError {
+    #[error("Server error: `{}'", _0)]
+    Server(#[source] MysqlError),
+}
```

Comment:
This is a partial copy of the `mysql_async::Error` enum.

Importing `mysql_async` here would break the Wasm compilation.

Reply:
Sure: https://github.com/prisma/prisma-engines/pull/4442/commits/bacb635367bb994939c9cdcf530033d334a3224b